### PR TITLE
Register Subdomain: click.node

### PR DESCRIPTION
**Pull request by node0505**
Added the domain: click.node

### CODE REVIEW

File: domains/click.node.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "node0505",
+        "email": "node0505@outlook.com"
+      },
+
+      "record": {
+        "CNAME": "cname.github.com"
```
Comment (WilliamDavidHarrison) [inline domains/click.node.json]: Invalid CNAME

**Review by wdhdev — DISMISSED** [review]
Nevermind, I didn't see the other commits.

**Review by wdhdev — CHANGES_REQUESTED** [review]
404

**Comment by node0505** [comment]
It's not a 404, it's a URL forwarder. Go to /google and it should resolve

**Review by wdhdev — DISMISSED** [review]
/google resolves. You should add a main page though.

**Comment by node0505** [comment]
Will do in the next few days 👍

**Pull request merged**